generic/src/main/java/org/datahem/processor/utils/Failure.java

```diff
@@ -0,0 +1,62 @@
+  package org.datahem.processor.utils;
+
+/*-
+ * ========================LICENSE_START=================================
+ * DataHem
+ * %%
+ * Copyright (C) 2018 - 2019 MatHem Sverige AB
+ * %%
+ * This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+ * =========================LICENSE_END==================================
+ */
+  
+import com.google.api.services.bigquery.model.TableFieldSchema;
+import com.google.api.services.bigquery.model.TableRow;
+import com.google.api.services.bigquery.model.TableSchema;
+import java.util.ArrayList;
+import java.util.List;
+
+  public class Failure{
+        private String target;
+        private String message;
+        private String error;
+        private String errorType;
```

Comment:
Perhaps the errorType could be an enum instead so that it's individual values can be directly referenced when using them? Will make using this class safer I think.

Reply:
Thanks